docs/projects/roman-numeral-conversion/2019-10-27-python.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Roman Numeral to Decimal number in Python
+layout: default
+last-modified: 2019-10-27
+featured-image: 
+tags: [python, roman_numeral_to_decimal_number]
+authors:
+  - auroq(code)
+  - rayavarapuvikram1
```

Comment:
Make sure you add yourself to https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml

Reply:
Done SIr.